deta/base.py

```diff
@@ -91,7 +89,35 @@ def _request(self, path: str, method: str, data: dict = None):
 
         if status in [200, 201, 207, 404]:
             return status, json.loads(payload) if status != 404 else None
-        raise urllib.error.HTTPError(url, status, res.reason, res.headers, res.fp)
+        raise urllib.error.HTTPError(
+            url, status, res.reason, res.headers, res.fp)
+
+class Drive(_Object):
+    def __init__(self, drive_name:str=None, project_key:str=None, project_id:str=None, host:str=None):
+        assert drive_name, "Please provide a Drive name. E.g 'mydrive"
+        assert project_key, "Please provide a project_key. Check docs.deta.sh"
+        self.name = drive_name
```

Comment:
things common to both `drive` and `base` should be in the parent class and you init the parent class with `super().__init__()`

Reply:
That makes sense! Thanks! :D